docs/book/features/block_volume.md

```diff
@@ -147,8 +148,8 @@ There are two types of volume provisioning in a Kubernetes cluster:
 
 ### Dynamically provision a block volume in Tanzu Kubernetes Grid Cluster<a id="dynamic_volume_provisioning_tanzu"></a>
 
-In `Tanzu Kubernetes Grid` cluster, the `StorageClass` gets automatically created and is made available to it by the
-underlying SV cluster. The name of this `StorageClass` is `gc-storage-profile` which looks like:
+In a Tanzu Kubernetes Grid cluster, the `StorageClass` gets automatically created and is made available to it by the
```

Comment:
I think `Project Pacific` is the old name. Supervisor cluster is more appropriate? https://docs.vmware.com/en/VMware-vSphere/7.0/vmware-vsphere-with-kubernetes/GUID-21ABC792-0A23-40EF-8D37-0367B483585E.html

Reply:
Changed it